### PR TITLE
Fixing links 2.5

### PR DESCRIPTION
**Pull request by dockerymick**
https://app.zenhub.com/workspaces/engineering-backlog-do-not-delete-604fab62d4b98d00150a2854/issues/stolostron/backlog/20294

### CODE REVIEW

File: about/quick_start.adoc [hunk 1]
```diff
@@ -27,9 +27,10 @@ If you experience problems, see the link:../troubleshooting/troubleshooting_intr
 . Before you install {product-title}, review the system configuration requirements and settings at link:../install/requirements.adoc#requirements-and-recommendations[Requirements and recommendations].
 Get information about required operating systems and supported browsers.
 For instance, you want to ensure that you have a supported {ocp} version so that you can set up your hub cluster.
+//not sure which file we are trying to reference here. There is no `prep` file. Should the user be sent to instlal/install_connected.adoc?
 . You also need to ensure that your hub cluster has the appropriate capacity.
 To prepare your hub cluster, see link:../install/prep.adoc#preparing-your-hub-cluster-for-installation[Preparing your hub cluster for installation].
```
Comment (dockerymick) [inline about/quick_start.adoc]: @swopebe @cadawson need confirmation about the correct reference link

Comment (swopebe) [reply about/quick_start.adoc]: Sorry, @dockerymick I will get to this later tonight or early morning...today was 1/2 day.

File: about/quick_start.adoc [hunk 2]
```diff
@@ -27,9 +27,8 @@ If you experience problems, see the link:../troubleshooting/troubleshooting_intr
...
 Get information about required operating systems and supported browsers.
 For instance, you want to ensure that you have a supported {ocp} version so that you can set up your hub cluster.
-. You also need to ensure that your hub cluster has the appropriate capacity.
-To prepare your hub cluster, see link:../install/prep.adoc#preparing-your-hub-cluster-for-installation[Preparing your hub cluster for installation].
-. With a supported version of {ocp-short} installed and running on your hub cluster, you can proceed with link:../install/install_connected.adoc#installing-while-connected-online[Installing while connected online].
+. You also need to ensure that your hub cluster is sized appropriately. See recommendations at link:../install/cluster_size.adoc#sizing-your-cluster[Sizing your cluster].
+. With a supported version of {ocp-short} installed and running on your hub cluster, you can proceed with link:../install/install_connected.adoc#installing-while-connected-online[Installing while connected online].
```
Comment (swopebe) [inline about/quick_start.adoc:31]: install/cluster_size.adoc is the new name of the file and links have been changed

**Comment by dockerymick** [comment]
Idk what happened but i reverted changes that the team made. Closing this PR and will reopen a new one

**Comment by dockerymick** [comment]
New pr: https://github.com/stolostron/rhacm-docs/pull/3443/files

**Pull request closed without merging**